lmnet/executor/train.py

```diff
@@ -38,7 +40,16 @@ def _save_checkpoint(saver, sess, global_step, step):
 def setup_dataset(config, subset, rank):
     DatasetClass = config.DATASET_CLASS
     dataset_kwargs = dict((key.lower(), val) for key, val in config.DATASET.items())
-    dataset = DatasetClass(subset=subset, **dataset_kwargs)
+    tfds_kwargs = dataset_kwargs.pop("tfds_kwargs", {})
+
+    # If there is a settings for TFDS, TFDS dataset class will be used.
```

Comment:
This changes need to be applied to  using `config.DATASET_CLASS` executable files also.

* `executor/tune_ray.py` 
* `executor/evaluate.py`
are waiting to be changed.

Reply:
Fixed!